### PR TITLE
Front end updates

### PR DESCRIPTION
**Pull request by mariom3**

### CODE REVIEW

File: portfolio/src/main/webapp/css/aboutme.css [hunk 1]
```diff
@@ -7,59 +7,60 @@ body {
...
 
 .custom-btn {
     border-radius: 0;
     border: 2px solid black;
     color: black;
 }
 
+#comments {
+  text-align: left;
+  font-weight: bold;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/css/aboutme.css:53]: nit:Order the properties alphabetically/.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -20,15 +20,18 @@ function getComments() {
...
   });
 }
 
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
-}
+function createCommentElement(userName, comment) {
+  const divElement = document.createElement('div');
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js]: Move the divElement close to where it is used, just above divElement.innerText.

**Review by sbahaud — APPROVED** [review]
LGTM with some minor comments.

Maybe update the PR description to say. "Updated styling and adding userName to the comment"

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (mariom3) [reply portfolio/src/main/webapp/script.js]: Done: 4c79013

File: portfolio/src/main/webapp/css/aboutme.css (hunk 1 above)
Comment (mariom3) [reply portfolio/src/main/webapp/css/aboutme.css:53]: Done: 979f7f6

**Pull request merged**